internal/services/services.go

```diff
@@ -35,85 +28,6 @@ const (
 	stopService
 )
 
-func listenToAutoPortForwardingChangesOnRemote(
```

Comment:
@mikeland86 any plans for this?

Reply:
let's kill it!

This was meant to auto-forward ports between devbox cloud and your local machine but in practice it's brittle compared to our existing service proxy.

Reply:
Okay.